.github/workflows/ci.yml

```
@@ -6,10 +6,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [2.7, "3.0", 3.1, 3.2]
+        ruby: [2.7, "3.0", 3.1, 3.2, 3.3, 3.4]
```

[SUGGESTION] Let's limit the core matrix to the supported ruby versions and a subset of the older ruby and activesupport version combinations to the `include`.

```suggestion
        ruby: ["3.2", "3.3", "3.4"]
```